app/assets/stylesheets/application.scss

```diff
@@ -129,6 +129,10 @@ h3,h4{
   }
 }
 
+h4 a{
```

Comment:
h5 (h4 has a lot of spacing) elements for titles, any better? I'll leave the 'clear' until #542 
![screen shot 2017-05-18 at 15 19 36](https://cloud.githubusercontent.com/assets/158833/26207175/b2c73dfe-3bde-11e7-8f1d-6c4dcc221ebc.png)